pkgs/top-level/all-packages.nix

```diff
@@ -357,7 +357,7 @@ with pkgs;
   };
 
   vcpkg-tool = callPackage ../by-name/vc/vcpkg-tool/package.nix {
-    fmt = fmt_10;
+    fmt = fmt_11;
```

Comment:
Could remove `vcpkg-tool` from `all-packages.nix` and use `fmt_11` in `package.nix` instead

Reply:
that changes the .override interface